VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationSelectView.java

```diff
@@ -96,21 +96,19 @@ public boolean isValidInput() {
 
         // check if this is required field and has an input value
         if (mField.isRequired() && !hasValue()) {
-            handleError(mField.getErrorMessage().getRequired());
+            String errorMessage = mField.getErrorMessage().getRequired();
+            if(errorMessage==null || errorMessage.isEmpty()){
+                errorMessage = getView().getResources().getString(R.string.error_select_field);
+                errorMessage = errorMessage.replaceFirst(getView().getResources()
```

Comment:
We should be using standard format strings for this instead of rolling our own formatting.